### ISSUE TITLE
Updating empty values

### ISSUE THREAD

**Issue by Guiqft** [rezo-labs/directus-extension-computed-interface#7]
If you clear the input of the original field, the computed interface may throw a error because `[].replace` cannot exists, for example. 
Or, if we clear a number field, the computed value may emit a "NaN" string, instead of null.

**Comment by duydvu** [comment]
Can you give an example to reproduce this bug?

**Comment by Guiqft** [comment]
Here is a GIF showing the bug:
![ezgif-5-e3eb47a73b](https://user-images.githubusercontent.com/9392803/197921332-d16c4163-c24a-47b9-8d49-1fc3c6b60cb6.gif)

I think that the computed value must emit a null if the source value is null.

**Comment by Guiqft** [comment]
We also have this:
![ezgif-1-5736797894](https://user-images.githubusercontent.com/9392803/197926213-17e12298-36ed-447a-b2d9-1f3f01b73534.gif)
I put these logs on `interface.vue:63`. Seems that the other computed field emitted a 0, his expression is simple: `{{ MULTIPLY(numero,2) }}`.
I think we must check if the source value is null before parse the expression.

**Comment by Guiqft** [comment]
@duydvu to reproduce this, you can create these fields:
- number (numeric)
- numberx2 (`{{ MULTIPLY(number, 2) }}`
- text (string)
- text_slug (`{{ SLUG(text) }}` 

When you edit the `text` field but leaves `number` empty and tries to save the item, the `numberx2` calculation is triggered and returns 0 (because `number(null) x 2 = 0`. Then, Directus complains about you trying to get out from a item that has unsaved changes.

**Comment by duydvu** [comment]
Nope, still can't see the bugs. Which Directus version are you using? Mine is 9.10.0

**Comment by Guiqft** [comment]
Hi @duydvu! Sorry for the long time. 
I can still reproduces this on a empty instance of Directus `10.5.3`. 
In the most basic example:
- Create a INT field `my_number` with simple input
- Create a INT field `my_number_x2` with the computed extension `{{ MULTIPLY(my_number, 2) }}`

When typing any number then erasing on the source input, the computed input gives me `0` as result, but I think it should be `null`. 

If the source is a empty value, the computed should be empty too. 
This is the same cause that makes the computed field starts with `0` on the video below ("Compute If Empty" is enabled). 


https://github.com/rezo-labs/directus-extension-computed-interface/assets/9392803/ca2f487d-fd71-4bd9-9266-1b3cc710040c
